packages/models-library/src/models_library/settings/docker_registry.py

```diff
@@ -0,0 +1,27 @@
+from typing import Optional
+
+from pydantic import BaseSettings, Field, SecretStr
+
+
+class RegistrySettings(BaseSettings):
+
+    auth: bool = Field(..., description="do registry authentication")
```

Comment:
FORMATTING: new style for settings  (discussed and approved in [#2369](https://github.com/ITISFoundation/osparc-simcore/pull/2369)) is to name the fields exactly as env vars and inherit from ``models_library.settings.base.BaseCustomSettings``

so this class (and any settings class capturing from env vars) should be
```python
class RegistrySettings(BaseCustomSettings): # <----
      REGISTRY_AUTH: bool = Field(..., description="...")
      REGISTRY_PATH
      REGISTRY_URL
      ...

```